js/app.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const all_animals = [];
+
+function Animal (horn){
+    this.image_url = horn.image_url;
+    this.title = horn.title;
+    this.description = horn.description;
+    this.keyword = horn.keyword;
+    this.horns = horn.horns;
+  all_animals.push(this);
+}
+
+Animal.prototype.render = function(){
+  const photo_section_html = $('#photo-template').html()
+
+  $('main').append('<section id="clone"></section>');
+  $('#clone').html(photo_section_html);
+  $('#clone').find('h2').text(this.name);
+  $('#clone').find('img').attr('src', this.image_url);
+  $('#clone').find('p').text(this.title).text(this.description);
+  $('#clone').attr('id', '');
+}
+
+const test_animal = new Animal({});
+test_animal.render();

```

Comment:
Line 25 and 26 are a great way to do a "proof-of-life" check. Once you know that is working and you've moved on to actually instantiating the Animal objects from the json file you should remove these lines. As it is now they will create an extra section on the index page that holds blank data (because the Animal object is blank), and this is no longer a useful check.